src/zoid/buttons/component.jsx

```diff
@@ -461,6 +461,12 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 }
             },
 
+            paymentRequest: {
+                type:       'object',
+                queryParam: true,
```

Comment:
@andyfen If this is approved, I'm not so sure we should have it passed as a queryParam.  Are we getting this from the query param anywhere?  We're trying to bring the query param down so it doesn't get truncated. We saw this occurring on native app switching.

Reply:
yup it is only accessed via xprops, will remove, great point.

Reply:
just to confirm `queryParam: false` we still have it available as a xprop it's just not in the queryparam right ?

Reply:
Correct